pkg/routeagent/controllers/route/route.go

```diff
@@ -348,7 +348,7 @@ func (r *Controller) createVxLANInterface(ifaceType int, gatewayNodeIP net.IP) e
 			name:     VxLANIface,
 			vxlanId:  100,
 			group:    gatewayNodeIP,
-			srcAddr:  vtepIP,
+			srcAddr:  nil,
```

Comment:
ahh, good one, to bad I missed this on the initial review.

Thanks for fixing it.